### PR TITLE
Add save dash with fake entry logic

### PR DESCRIPTION
**Pull request by mournfulCoroner**
Prev PR: https://github.com/datalens-tech/datalens-ui/pull/42
In this PR, the logic of creating a fake entry has been added

### CODE REVIEW

File: src/ui/units/dash/store/utils.ts [hunk 1]
```diff
@@ -0,0 +1,51 @@
...
+
+import {Mode} from '../modules/constants';
+
+const storeI18n = I18n.keyset('dash.store.view');
+const dashCreateI18n = I18n.keyset('component.dialog-create-dashboard.view');
+
+export const getFakeDashEntry = (workbookId?: string) => {
+    const salt = Math.random().toString();
+    const {counter, id: newTabId} = generateUniqId({salt, counter: 0, ids: []});
+    const initialKey = `${DL.USER_FOLDER}${dashCreateI18n('label_default-name')}`;
```
Comment (mournfulCoroner) [inline src/ui/units/dash/store/utils.ts:14]: Using the same i18n key as the initial name in the dashboard creation modal (`DialogCreateDashboard`)

**Review by Marginy605 — CHANGES_REQUESTED** [review]

File: src/ui/components/EntryDialogues/DialogCreateDashboard/DialogCreateDashboard.tsx [hunk 2]
```diff
@@ -65,21 +72,23 @@ class DialogCreateDashboard extends React.Component<Props> {
...
+                data,
             },
         });
     };
 
     private onApply = async (key: string) => {
-        const data = await this.props.sdk.charts.createDash({
-            data: {key},
+        const {data} = this.props;
+        const responce = await this.props.sdk.charts.createDash({
```
Comment (Marginy605) [inline src/ui/components/EntryDialogues/DialogCreateDashboard/DialogCreateDashboard.tsx:87]: type

File: src/ui/units/dash/components/DashActionPanel/DashActionPanel.tsx [hunk 3]
```diff
@@ -81,9 +78,15 @@ export type ActionPanelProps = OwnProps & StateProps & DispatchProps;
 type ActionPanelState = {};
 
 class DashActionPanel extends React.PureComponent<ActionPanelProps, ActionPanelState> {
+    isFakeEntry = Boolean(
```
Comment (Marginy605) [inline src/ui/units/dash/components/DashActionPanel/DashActionPanel.tsx:81]: Problem case:
- open new dash
- open empty created dash
Expect: updated actionPanel in top
Reality: old actioPanel

**Review by Marginy605 — CHANGES_REQUESTED** [review]

File: src/ui/units/dash/components/DashActionPanel/DashActionPanel.tsx (hunk 3 above)
Comment (mournfulCoroner) [reply src/ui/units/dash/components/DashActionPanel/DashActionPanel.tsx]: Getter solved the problem

**Review by Marginy605 — APPROVED** [review]

**Pull request merged**